code/modules/projectiles/projectile/energy.dm

```diff
@@ -67,7 +67,7 @@
 /obj/item/projectile/energy/electrode/stunshot
 	nodamage = 0
 	damage = 10
-	agony = 50
+	agony = 65

```

Comment:
@JakeDaBoss The energy gun and taser don't _use_ /obj/item/projectile/energy/electrode/stunshot.